src/App.js

```diff
@@ -1,13 +1,7 @@
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] You are yet to make your app component class-based as requested earlier. Kindly do that.